src/likelihoods/poisson.jl

```diff
@@ -6,6 +6,11 @@ end
 
 const AugPoisson = PoissonLikelihood{<:ScaledLogistic}
 
+logλ(l::AugPoisson) = log(l.invlink.λ)
```

Comment:
is it worth having this as a separate function?
you also have `lik.invlink.λ` (no log) in `aux_prior`, and with log in `expected_logtilt`.
I'm wondering if it might be more helpful to define a helper function that just returns `l.invlink.λ`?

Reply:
Could be but I will leave it to another time :) It's not so important.